### DISCUSSION TITLE
How to Export Multimedia

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#315] [HOW TO...]
**Used By:** ALL
**In Use?:** Yes

There are a several ways that you can bulk export multimedia files:

1. 'Multimedia Export' Report -- for Main media files
2. 'Supplementary Export' Report -- for files on the Supplementary tab
3. 'Export Tool’ -- for Main + Supplementary media files	

[Googledoc Workflow](https://docs.google.com/document/d/1TGW4iffHOfdqYUJ7qFZy-YjcUxKQWVDYT9Vyieo4er0)

**Comment by magpiedin** [comment]
(corresponding FAQ: https://github.com/fieldmuseum/EMu-Documentation/discussions/360)
